### PR TITLE
BLOCKCHAIN-60 Replace Apollo with redux-saga + axios in frontend

### PR DESCRIPTION
**Pull request by MichalLustykNeti**

### CODE REVIEW

File: src/api/explorer.js [hunk 1]
```diff
@@ -22,46 +22,136 @@ const receiptsQuery = `
...
+    receipt_id: r.receiptId,
+  }));
+
+  return receipts.reduce((o, r) => ({ ...o, [r.receipt_id]: r }), {});
+};
+
+export const getDollarsTransfers = async (substrate_address) => {
+  const result = await getApi().post('/graphql', {
+    query: `
+        query Transfers($orderBy: [TransfersOrderBy!], $filter: TransferFilter) {
```
Comment (kacperzuk-neti) [inline src/api/explorer.js:66]: Either move this query out to a variable at the top of file or inline the `receiptsQuery` - I don't care either way, but let's be consistent.

File: src/api/explorer.js [hunk 2]
```diff
@@ -22,46 +22,136 @@ const receiptsQuery = `
...
+  const transfers = result.data?.data?.query?.transfers?.nodes;
+  if (!transfers) return [];
+
+  return transfers.map((n) => ({ asset: 'LLD', ...n }));
+};
+
+export const getMeritsTransfers = async (substrate_address) => {
+  const result = await getApi().post('/graphql', {
+    query: `
+        query Merits($orderBy: [MeritsOrderBy!], $filter: MeritFilter) {
```
Comment (kacperzuk-neti) [inline src/api/explorer.js:114]: Either move this query out to a variable at the top of file or inline the `receiptsQuery` - I don't care either way, but let's be consistent.

**Review by kacperzuk-neti — CHANGES_REQUESTED** [review]

**Comment by kacperzuk-neti** [comment]
Fix commit msg.

File: src/api/explorer.js [hunk 3]
```diff
@@ -22,46 +22,140 @@ const receiptsQuery = `
...
-    if (pallet == "ethLLMBridge") {
-        return "LLM";
-    } else if (pallet == "ethLLDBridge") {
-        return "LLD";
-    } else {
-        throw new Error("Unknown pallet");
+const getMeritsTransfersQuery = `
+  query Transfers($orderBy: [TransfersOrderBy!], $filter: TransferFilter) {
+    query {
+      transfers(orderBy: $orderBy, filter: $filter) {
```
Comment (kacperzuk-neti) [inline src/api/explorer.js:50]: This fetches `LLD` transfers, not `LLM` transfers. LLM transfers are called `Merits` in GraphQL schema.

**Review by kacperzuk-neti — CHANGES_REQUESTED** [review]

**Review by DorianSternVukotic — APPROVED** [review]

**Pull request merged**